app/components/leagueStandingTable/leagueStandingTable.tsx

```diff
@@ -0,0 +1,37 @@
+import { Table } from "../baseComponents";
+import { getWikipediaContestantDataFetcher } from "@/app/utils/wikiFetch";
+import generateListOfContestantRoundLists from '@/app/generators/contestantRoundListGenerator'
+import { getCompetingEntityList } from "@/app/utils/wikiQuery"
+
+import styles from "./leagueStandingTable.module.scss";
+
+export default async function LeagueStandingTable({ contestantsScores }:{contestantsScores: any}){
+    type rowState = any[];
+    const tableColNames: rowState = ["Rank", "Name", "Score"];
+    const tableRows: rowState = [];
+    const tableData = {
+        colsNames: tableColNames,
+        rows: tableRows
+    }
+
+    contestantsScores.forEach((contestantData:any) =>{
+        const { content: { props }} = contestantData;
+        const contestantRoundScores = props.contestantRoundScores;
+        const contestantRoundTotalScore = contestantRoundScores.at(-1).contestantRoundData[0].totalScore;
+        const tableContestantData = [props.contestantName, contestantRoundTotalScore]
+        tableData.rows.push(tableContestantData);
+    });
+
+    tableData.rows.sort((a, b) => {
+        const aScore = a.filter((param:number|string) => typeof param === "number")[0];
+        const bScore = b.filter((param:number|string)=> typeof param === "number")[0];
+        const scoreToReturn = aScore > bScore ? -1 : 1;
+        return scoreToReturn;
+    });
+
+    tableData.rows.map((tableRow, index) => {
+        return tableRow.unshift(index+1);
+    })
```

Comment:
I don't suppose I can convince you that this is a concern of the display layer and so it should go in the component itself 😅